test/markup/php/functions.txt

```diff
@@ -49,3 +49,15 @@ setAlarm(
     label: 'foo',
```

Comment:
Maybe you want to add a test that tests a comment with a apostrophe?

Reply:
> Maybe you want to add a test that tests a comment with a apostrophe?

The single quotes or apostrophes are irrelevant to the issue, they were merely a symptom. Since the text was NOT wrapped as a comment, when Highlight.js encountered the single quote it would start a string section which changed the color. That surfaced the root cause: the text was not considered a comment but some inline PHP.